coverage/cmdline.py

```diff
@@ -227,7 +229,15 @@ class Opts:
         "", "--version", action="store_true",
         help="Display version information and exit.",
     )
-
+    dump_signal = optparse.make_option(
+        '', '--dump_signal', action='store', metavar='DUMP_SIGNAL',
```

Comment:
```suggestion
        '', '--dump-signal', action='store', metavar='DUMP_SIGNAL',
```
Options use hyphens, not underscores.